glnx-fdio.c

```diff
@@ -822,12 +845,9 @@ glnx_file_copy_at (int                   src_dfd,
       goto out;
     }
 
-  src_fd = TEMP_FAILURE_RETRY (openat (src_dfd, src_subpath, O_RDONLY | O_CLOEXEC | O_NOCTTY | O_NOFOLLOW));
-  if (src_fd == -1)
-    {
-      glnx_set_error_from_errno (error);
-      goto out;
-    }
+  /* Possibly consider O_NOFOLLOW here? Not sure it's worth it.*/
```

Comment:
Or make it another arg to the function? Seems like changing this behaviour has the potential to introduce some subtle regressions.